arm/cdn/customdomains.go

```diff
@@ -118,14 +122,18 @@ func (client CustomDomainsClient) CreateResponder(resp *http.Response) (result a
 	return
 }
 
-// DeleteIfExists sends the delete if exists request.
+// DeleteIfExists sends the delete if exists request.This method handles
```

Comment:
@garimakhulbe It still isn't very clear to me, and probably will be confusing to the users as well.

What is the primary use case of this `chan`:

a. disable polling
b. cancel polling

if (a) I believe user already has ways to disable polling by reaching out to autorest.Client and flipping a flag there. if (b), then the documentation still doesn't make it clear that it's primarily for "cancellation" and not "opting out".